cpp/ps0005/ps0005_sol.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+
+int main()
+{
+    // prompting user for 2 operands and an operator 2/2
+    double number1,number2,result;
+    int result1;
+    char op;
+   
+    std::cout<<"please Enter the 2 oprants\n";
+    std::cin>>number1>>number2;
+
+    std::cout<<"please Enter the oprerators\n";
+    std::cin>>op;
```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input